drivers/via/server.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/byuoitav/av-control-api/drivers"
+	"github.com/byuoitav/kramer-driver"
+	"github.com/spf13/pflag"
+)
+
+// imports
+
+func main() {
+	var (
+		port     int
+		username string
+		password string
+	)
+
+	pflag.IntVarP(&port, "port", "P", 80, "port to run the server on")
+	pflag.StringVarP(&username, "username", "u", "", "username for device")
+	pflag.StringVarP(&password, "password", "p", "", "password for device")
+	// other flags
+
+	pflag.Parse()
+
+	// create a net.Listener to run the server on
+	addr := fmt.Sprintf(":%d", port)
+	lis, err := net.Listen("tcp", addr)
+	if err != nil {
+		fmt.Printf("failed to start server: %s\n", err)
+		os.Exit(1)
+	}
+
+	// import driver library
+	createDsp := func(ctx context.Context, addr string) (drivers.DSP, error) {
+		return &kramer.Via{
+			Address:  addr,
+			Username: username,
+			Password: password,
+		}, nil
```

Comment:
Let's add a default logger in here so we get some logs out of the microservice. 
```suggestion
		logger := drivers.Log.Named(addr)
		return &kramer.Via{
			Address:  addr,
			Username: username,
			Password: password,
			Logger: logger,
		}, nil
```